src/components/Card/index.stories.tsx

```diff
@@ -23,6 +24,12 @@ const Grid = styled.div`
   }
 `
 
+const Header = styled.div`
```

Comment:
Ideally, this should be a `header` element, and the `leftAdornment` inside the Card should be a regular div. Not sure if it makes sense that the left adornment is a header 😄